src/pages/JobDescription/JobDescriptionPage.jsx

```diff
@@ -0,0 +1,75 @@
+import { Box, Button, Grid, Typography } from "@mui/material/";
+import { useState } from "react";
+import { v4 as uuidv4 } from "uuid";
+import PageLayout from "../../components/PageLayout";
+import JobDescriptionItem from "./components/JobDescriptionItem";
+import mockedSkills from "./mockData";
+/**
+ * A component which renders the JobDescriptionPage to let the company select the necessary skills for the job
+ * @returns {JSX.Element} The JobDescriptionPage component.
+ */
+
+function JobDescriptionPage() {
+  const [skills, setSkills] = useState(
+    mockedSkills.reduce((obj, item) => {
+      obj[item] = false;
+      return obj;
+    }, {})
+  );
```

Comment:
Why did you use reduce?

Reply:
The reduce method is used here instead of map because it transforms an array (mockedSkills) into an object, accumulating each array item as a key with a value of false, which is not achievable with map that only alters each array element but maintains the array structure.

Reply:
Thank you